src/main/java/org/zone/Permissions.java

```diff
@@ -9,7 +9,25 @@
     REGION_ADMIN_INFO("zone.region.admin.info"),
     BYPASS_INTERACTION_DOOR("zone.bypass.interaction.door"),
     BYPASS_INTERACTION_BLOCK("zone.bypass.interaction.block"),
-    REGION_EDIT_BOUNDS("zone.region.edit.bounds");
+    REGION_EDIT_BOUNDS("zone.region.edit.bounds"),
+    FLAG_SET("zone.region.flag.set"),
```

Comment:
This should be less generic. Whats the flag type?

Reply:
eh idk this is yours?

Reply:
and it's a command

Reply:
remove the flag set then ;)